data_validation/data_validation.py

```diff
@@ -357,3 +359,11 @@ def combine_data(self, source_df, target_df, join_on_fields):
                 rsuffix=consts.OUTPUT_SUFFIX,
             )
         return df
+
+    def _adjust_schema(self,data_frame):
```

Comment:
You may not need to do this since PR #392 implements a fix within the Ibis client so that the execute query will not append a 't0' to the column names.

Reply:
removed the above function.